### PR TITLE
docs: fix indentation issue in component lifecycle

### PR DESCRIPTION
**Pull request by cadedupont**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #50488
Tab size differs in code snippet from 2 to 4, modified to make indentation size 4 for entire snippet.

## What is the new behavior?
Code indentation is more evenly distributed in lifecycle guide documentation

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/lifecycle-hooks.md [hunk 1]
```diff
@@ -108,8 +108,8 @@ Register a callback with the `DestroyRef`:
 ```ts
 @Component(...)
 class Counter {
-  count = 0;
-  constructor() {
+	count = 0;
+	constructor() {
```
Comment (JoostK) [inline aio/content/guide/lifecycle-hooks.md:112]: You'll want to make the opposite change: replace the tabs in the lines below with 2 spaces each.

**Review by JoostK — COMMENTED** [review]
Please see [CONTRIBUTING.md](https://github.com/angular/angular/blob/6f5dabe0d25a5660b7c3001041449b4622dd8924/CONTRIBUTING.md#-submitting-a-pull-request-pr) for some instructions on how the PR can be updated.

**Comment by JoostK** [comment]
The changes now look good, but you'll need to combine the two commits into one and rephrase the commit message to align with the convention.

You can either run an interactive rebase to `edit` the first commit and `fixup` the second commit, or perform a soft reset to before the commits were made and then create a commit from scratch. The latter is probably easiest, although learning how to do rebases could be beneficial in the long run 😃 

Using a soft reset looks something like this:
```bash
git reset --soft HEAD~2
git commit -am 'docs: fix indentation issue in component lifecycle'
git push --force-with-lease
```

**Comment by cadedupont** [comment]
3rd time's the charm hopefully haha, apologies but thank you for the guidance! :)

**Review by JoostK — APPROVED** [review]
Thanks for the fix!

**Review by crisbeto — APPROVED** [review]
LGTM

**Comment by alxhub** [comment]
This PR was merged into the repository by commit ab35596b36da1e1029c0d9c657277400d42cf6a2.

**Pull request closed without merging**